src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -0,0 +1,526 @@
+import app = require("durandal/app");
+import appUrl = require("common/appUrl");
+import database = require("models/resources/database");
+import viewModelBase = require("viewmodels/viewModelBase");
+import accessManager = require("common/shell/accessManager");
+import eventsCollector = require("common/eventsCollector");
+import jsonUtil = require("common/jsonUtil");
+import getDatabaseSettingsCommand = require("commands/database/settings/getDatabaseSettingsCommand");
+import saveDatabaseSettingsCommand = require("commands/database/settings/saveDatabaseSettingsCommand");
+import virtualGridController = require("widgets/virtualGrid/virtualGridController");
+import hyperlinkColumn = require("widgets/virtualGrid/columns/hyperlinkColumn");
+import textColumn = require("widgets/virtualGrid/columns/textColumn");
+import saveDatabaseSettingsConfirm = require("viewmodels/database/settings/saveDatabaseSettingsConfirm");
+import models = require("models/database/settings/databaseSettingsModels");
+import popoverUtils = require("common/popoverUtils");
+import genUtils = require("common/generalUtils");
+
+type viewModeType = "summaryMode" | "editMode";
+
+class categoryInfo {
+    name = ko.observable<string>();
+    customizedEntriesCount = ko.observable<number>();
+    
+    constructor (categoryName: string, numberOfCustomizedEntries: number) {
```

Comment:
remove space

Reply:
seems there is just one space after constructor....